### DISCUSSION TITLE
Export dashboards to external website without iframe

### DISCUSSION THREAD

**Discussion by ciolo** [apache/superset#19707] [Q&A / Help]
Hi everyone,
I exported some dashboards of my superset instance in an external website using an iframe with flag `?standalone=true` successfully. Everything works fine. 
But my question is: is it possibile export dashboard without using iframe? I've seen the `superset-ui` package but it seems no longer maintained. Am I wrong? If so, how can I export dashboards without using an iframe, for example by integrating everything in JS?

Thank u.

**Comment by srinify** [comment]
Moving to Discussions so hopefully someone can chime in! Issues is focused on bugs!

**Comment by mcftc** [comment]
year 2025 still does not have built in feature like this but its possible of course.
